### PR TITLE
This is freeze header for module for maps /editor

### PR DESCRIPTION
**Pull request by ernestweems**
@adamdoe  please see commit

### CODE REVIEW

File: packages/core/components/DataTable/DataTable.tsx [hunk 1]
```diff
@@ -125,6 +125,16 @@ const DataTable = (props: DataTableProps) => {
     maxHeight: config.table.limitHeight && `${config.table.height}px`,
     OverflowY: 'scroll'
   }
+  const [height, getTableHeight] = useState(0)
```
Comment (adamdoe) [inline packages/core/components/DataTable/DataTable.tsx:128]: These three react hooks shouldn't be called conditionally. Use a more descriptive variable here, `height` could refer to a number of different heights throughout the app. Change `getTableHeight` to `setTableHeight` to follow React conventions.

File: packages/core/components/DataTable/DataTable.tsx [hunk 2]
```diff
@@ -125,6 +125,16 @@ const DataTable = (props: DataTableProps) => {
...
+  const tableHgt = useRef(null)
+  useEffect(() => {
+    getTableHeight(tableHgt.current.clientHeight)
+  }, [])
+
+  const fixedHeight = {
```
Comment (adamdoe) [inline packages/core/components/DataTable/DataTable.tsx:134]: The ref is always returning zero for me here, so the table is not displaying for me.

File: packages/core/components/DataTable/DataTable.tsx [hunk 3]
```diff
@@ -125,6 +125,16 @@ const DataTable = (props: DataTableProps) => {
     maxHeight: config.table.limitHeight && `${config.table.height}px`,
     OverflowY: 'scroll'
   }
+  const [height, getTableHeight] = useState(0)
+  const tableHgt = useRef(null)
```
Comment (adamdoe) [inline packages/core/components/DataTable/DataTable.tsx:129]: Best to spell out `tableHeight`. Its a few extra characters but will avoid confusion when other developers work on the app later on.

**Review by adamdoe — CHANGES_REQUESTED** [review]

File: packages/core/components/DataTable/DataTable.tsx (hunk 1 above)
Comment (ernestweems) [reply packages/core/components/DataTable/DataTable.tsx:128]: ok will reach in Teams

**Pull request closed without merging**